### DISCUSSION TITLE
[MSW] Warning: intercepted a request without a matching request handler:   • GET /config/config.json

### DISCUSSION THREAD

**Discussion by riksbyggarn** [mswjs/msw#2081] [Q&A]
I have my base_url in config.json i think it cant find it, how could i get the baseurl?

**Comment by kettanaito** [comment]
Hi. I don't quite understand what you are asking. Can you please rephrase your question, include some code examples (your MSW setup, your handlers), and describe what you are trying to achieve?

The discussion title refers to a warning MSW prints when it detects a request you don't have a request handler for. Is that causing trouble? Do you wish to ignore this warning for the `/config/config.json` request? 

I'm happy to help but first you need to [Ask a question](https://kettanaito.com/blog/how-to-ask-questions).

**Reply by frst-mg** [reply]
The problem is i have my base_url in a config.json so it seems like it cant find it, anyway in my handler if i for example do /api/tests and use that in my test file and do a HttpResponse.error() it triggers my error message, but then if i change the url to /api/whatever it still trigger the error, dosen't the url in the test match the exact url in the handler?
